src/viteAPI/provider.ts

```diff
@@ -12,6 +12,7 @@ class ProviderClass {
     private requestList: {[id:number]:()=>void} = {};
     private requestId = 0;
 
+
     constructor(provider: any, onInitCallback: Function) {
```

Comment:
I prefer a more descriptive type than `any`

Reply:
As above said, these should be a new PR to do.